snaps/learn/tutorials/transaction-insights.md

```diff
@@ -32,70 +38,493 @@ starter kit by running:
+    "@metamask/eslint-config": "^12.2.0",
+    "@metamask/eslint-config-jest": "^12.1.0",
+    "@metamask/eslint-config-nodejs": "^12.1.0",
+    "@metamask/eslint-config-typescript": "^12.1.0",
+    "@metamask/snaps-cli": "^6.1.0",
+    "@metamask/snaps-jest": "^7.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.42.1",
+    "@typescript-eslint/parser": "^5.42.1",
+    "eslint": "^8.45.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-import": "~2.26.0",
+    "eslint-plugin-jest": "^27.1.5",
+    "eslint-plugin-jsdoc": "^41.1.2",
+    "eslint-plugin-n": "^15.7.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-promise": "^6.1.1",
+    "jest": "^29.5.0",
+    "prettier": "^2.7.1",
+    "prettier-plugin-packagejson": "^2.2.11",
+    "rimraf": "^3.0.2",
+    "ts-jest": "^29.1.0",
+    "typescript": "^4.7.4"
+  },
+  "packageManager": "yarn@3.2.1",
+  "engines": {
+    "node": ">=18.6.0"
+  },
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://registry.npmjs.org/"
+  }
+}
+
+```
+</TabItem>
+</Tabs>
+
+
+#### 2.2. Update your Snap's name
+
+Update your Snap's name in MetaMask.
+MetaMask uses the `proposedName` of the Snap, currently "TypeScript Example" in the template.
+
+Open `packages/snap/snap.manifest.json` in a text editor.
+Edit the `"proposedName"` property within the metadata to provide a functional name such as "Transaction insights":
+
+<Tabs>
+  <TabItem value="Line updates" label="Update snap.manifest.json" default>
+
+```json title="snap.manifest.json"
+{
+  "version": "0.1.0",
+  "description": "An example Snap written in TypeScript.",
+  "proposedName": "Transaction insights",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/template-snap-monorepo.git"
+  },
+```
 
-In particular, the Snap shows the user the percentage of gas fees they would pay for their transaction.
-It gets the current gas price by calling the
-[`eth_gasPrice`](/wallet/reference/eth_gasPrice) RPC
-method using the global Ethereum provider made available to Snaps.
+</TabItem>
+<TabItem value="Full page" label="Example" default>
+
+```json title="snap.manifest.json"
+{
+  "version": "0.1.0",
+  "description": "An example Snap written in TypeScript.",
+  "proposedName": "Transaction insights",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/template-snap-monorepo.git"
+  },
+  "source": {
+    "shasum": "9UBmDB7B0fFYh9tYoj8+NEN85MfqhkQQRc1MujXdq8w=",
+    "location": {
+      "npm": {
+        "filePath": "dist/bundle.js",
+        "packageName": "snap",
+        "iconPath": "images/gas.svg",
+        "registry": "https://registry.npmjs.org/"
+      }
+    }
+  },
+   "initialPermissions": {
+    "snap_dialog": {},
+    "endowment:rpc": {
+      "dapps": true,
+      "snaps": false
+    },
+    "endowment:network-access": {}
+  },
+  "manifestVersion": "0.1"
+}
+```
+</TabItem>
+</Tabs>
+
+#### 2.3. Update your Snap's button
```

Comment:
I'm confused, can you clarify your reason for keeping this step? I find it confusing to ask users to update the text of a button that is non-functional.

We explain in the "Test your Snap" step that the button is non-functional, and in "Next steps" we suggest removing the button altogether.

If you prefer to keep something here about the button, we should guide users to remove the button instead of updating its text.

Reply:
We can probably prevent the div from being activated -- that wasn't in my wheelhouse: can investigate

Reply:
OK -- found a simple way to disable